helm_deploy/hmpps-staff-lookup-service/values.yaml

```diff
@@ -34,6 +34,7 @@ generic-service:
       OAUTH_CLIENT_ID: "AUTH_API_CLIENT_ID"
       OAUTH_CLIENT_SECRET: "AUTH_API_CLIENT_SECRET"
       OAUTH_TENANT_ID: "AUTH_API_TENANT_ID"
+      EMAIL_DOMAIN: "EMAIL_DOMAIN"
```

Comment:
I think it is fine to hard-coded `justice.gov.uk` in the `application.yaml` .. or even just have it as a default.

Putting it in the secrets makes it harder to know what it is - and eventually there might be more.